modules/mongodb/src/main/java/org/testcontainers/containers/MongoDBContainer.java

```diff
@@ -20,6 +20,7 @@
     private static final int AWAIT_INIT_REPLICA_SET_ATTEMPTS = 60;
     private static final String MONGODB_VERSION_DEFAULT = "4.0.10";
     private static final String MONGODB_DATABASE_NAME_DEFAULT = "test";
+    private String databaseName = MONGODB_DATABASE_NAME_DEFAULT;
```

Comment:
since it is not used anywhere but `getReplicaSetUrl`, WDYT about removing the field and adding an overloaded `getReplicaSetUrl(String)` method?

Reply:
It might be puzzling for users who are used to Testcontainers’ `with…` methods to create containers. However, given the rare case when a user needs to specify a MongoDB database name for testing, we can go with your solution, I’ll adjust changes.

Reply:
@bsideup done, please, check